index.html

```diff
@@ -370,14 +372,11 @@
 ,{"system":"Databend","date":"2023-02-26","machine":"c6a.metal, 500gb gp2","cluster_size":1,"comment":"","tags":["Rust","column-oriented","MySQL compatible","ClickHouse derivative"],"load_time":70,"data_size":20922561953,"result":[[0.073969,0.002969,0.002275],[0.289053,0.0145,0.013984],[0.597566,0.020848,0.022175],[1.241144,0.019701,0.019218],[1.407042,0.266073,0.245832],[1.707674,0.265472,0.272181],[0.037721,0.015296,0.014786],[0.044271,0.019238,0.018279],[1.814285,0.29248,0.287474],[3.034563,0.304214,0.31072],[1.634329,0.286281,0.299248],[1.937083,0.177104,0.188476],[1.52924,0.28984,0.289497],[3.075821,0.461813,0.428307],[1.708843,0.27206,0.275821],[0.83204,0.234342,0.218837],[3.085068,0.494668,0.480146],[2.891979,0.477079,0.457178],[6.122974,0.690913,0.634832],[0.186429,0.004711,0.00394],[10.868874,0.099969,0.086985],[13.480312,0.11198,0.10336],[24.405612,0.431874,0.440055],[10.061871,0.2325,0.242135],[0.647733,0.046068,0.039222],[2.100647,0.046711,0.043591],[1.391526,0.041194,0.039617],[11.536597,0.093705,0.088706],[9.839564,1.566094,1.5916],[0.16243,0.049029,0.049039],[5.631392,0.26343,0.225562],[7.261623,0.304319,0.279184],[5.646363,1.06771,0.910199],[11.17362,0.789665,0.684581],[11.251824,0.766817,0.682614],[0.663092,0.261053,0.272992],[0.40079,0.083889,0.091102],[0.073341,0.021988,0.028371],[0.091126,0.019851,0.013461],[0.151101,0.058686,0.058112],[0.281109,0.016812,0.014878],[0.398818,0.015711,0.022669],[0.222298,0.011058,0.014887]],"source":"databend/results/c6a.metal.json"}
-,{"system":"DataFusion (single parquet)","date":"2023-04-11","machine":"c6a.4xlarge, 500gb gp2","cluster_size":1,"comment":"v22.0.0 (34c9bce)","tags":["Rust","column-oriented","embedded","stateless"],"load_time":0,"data_size":14779976446,"result":[[2.646,0.225,0.226],[0.099,0.079,0.079],[0.173,0.138,0.139],[0.349,0.126,0.125],[2.79,6.718,2.717],[1.911,1.753,1.77],[0.09,0.079,0.076],[0.106,0.081,0.081],[3.032,3.054,3.675],[3.634,3.564,3.647],[0.508,0.397,0.402],[0.529,0.424,0.423],[2.1,2.057,2.04],[4.415,4.169,3.707],[2.198,2.143,2.08],[4.043,3.516,5.126],[5.065,6.106,8.249],[4.524,4.42,4.347],[10.981,11.07,11.203],[0.348,0.112,0.116],[9.968,1.594,1.619],[11.178,1.917,1.9],[22.12,4.337,4.352],[56.098,12.159,12.128],[2.581,0.582,0.583],[0.756,0.478,0.475],[2.556,0.604,0.57],[9.592,2.525,2.479],[9.067,5.925,5.927],[0.638,0.563,0.585],[3.278,3.119,2.94],[7.71,4.401,4.398],[null,null,null],[12.566,8.695,8.954],[12.827,9.199,11.148],[3.623,3.514,3.526],[0.506,0.413,0.406],[0.245,0.217,0.214],[0.249,0.194,0.194],[0.892,0.772,0.774],[0.175,0.107,0.106],[0.116,0.095,0.093],[0.142,0.112,0.117]],"source":"datafusion/results/single.json"}
```

Comment:
hm... looks like `datafusion/results/single.json` is removed from `index.html`, investigating

Reply:
fixed at 92689ce